install_requirements.py

```diff
@@ -0,0 +1,153 @@
+import os
```

Comment:
Please add a copyright/license header. This is largely copied from the Meta-written file so it at least deserves the Meta copyright header, but since you wrote the file you can also add your own copyright line (for yourself or your company, depending on how you signed the CLA) if you'd like to.

```suggestion
#!/usr/bin/env python3
# Copyright (c) Meta Platforms, Inc. and affiliates.
# All rights reserved.
#
# Copyright <YOUR-COPYRIGHT-IF-YOU-WANT-ONE>.
#
# This source code is licensed under the BSD-style license found in the
# LICENSE file in the root directory of this source tree.

import os
```

And may as well add the `#!/usr/bin/env python3` line. Won't be correct if `python` is the proper binary, but it will help some people.